guide/how-it-works/sign-in-with-bitcoin.md

```diff
@@ -0,0 +1,161 @@
+   alt-text = "Hand cursor hovering over a big 'Sign in' button"
+   caption = ""
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Sign in with bitcoin
+{:.no_toc}
+
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
+Identity is at the core of bitcoin, as it is a requirement to create what we refer to as ownership. Bitcoin is assigned to addresses. Addresses are generated from public-private key pairs. Owning bitcoin means being solely in control of these private keys. See [public-key cryptography](https://en.wikipedia.org/wiki/Public-key_cryptography)) for more information on this topic.
+
+The cryptography involved in this can also be used to generate public keys for other purposes, like a unique identifier that websites and applications can use for their account systems. This is similar to approaches like [Passkeys](https://developer.apple.com/passkeys/), which also look at replacing email & password combos with unique identifiers. [Decentralized identities](https://en.wikipedia.org/wiki/Decentralized_identifier) (DID) are another emerging approach.
+
+In the context of bitcoin, this allows users to sign in to services with their wallets and also use this connection to make low-friction payments.
+
+{% include tip/open.html color="blue" icon="info" label="All eyes on lightning" %}
+
+On this page, we only focus on lightning wallets, as they lend themselves much better to this type of user experience.
+
+{% include tip/close.html %}
+
+#### Support
+{:.no_toc}
+
+“Sign in with bitcoin” is not widely supported. It uses new technologies like [WebLN](https://webln.dev/#/) and [LNURL-auth](https://github.com/fiatjaf/lnurl-rfc/blob/luds/04.md) that are still maturing and looking for broader adoption. To try this out, you can use the [Alby](https://getalby.com/) browser extension wallet with a service like [Stacker News](https://stacker.news/) that allows you to spend satoshi to upvote news stories (more lightning apps can be found in the [bolt.fun maker directory](https://makers.bolt.fun/products)).
+
+### Wallet-service communication
+
+Your service and the user wallet need to be able to communicate with each other to exchange information. There are two basic ways to handle this—direct and bridged.
+
+#### Direct communication
+
+The smoothest user experience is when the wallet and the service can directly communicate with each other. For example, a browser wallet and a website can send messages back and forth (see [WebLN](https://www.webln.guide/introduction/readme)). This allows for convenient features like allowances for automatic approval of payments for services the user trusts.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/direct-connection.png"
+   retina = "/assets/images/guide/how-it-works/sign-in-with-bitcoin/direct-connection@2x.png"
```

Comment:
Wasn't clear what is going on in this picture, is this mean the user doesn't have to log in to communicate with the web app? some things that could this section clearer.

- Caption on image explaining what is going on
- An example of a direct communication app so people can go test it out themselves
- An action in the image that the user is taking

Reply:
This paragraph is not about specific actions. It illustrates how the wallet and the service communicate with each other. Is the text above not clear enough?

Reply:
I added captions to all images. Hope that clarifies the focus.